src/styles/styles.js

```diff
@@ -494,6 +548,68 @@ const styles = {
         textAlignVertical: 'center',
     },
 
+    expensiPickerContainer: {
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        borderColor: themeColors.border,
+        justifyContent: 'center',
+        backgroundColor: '#fff',
+    },
+    expensiPickerContainerOnFocus: {
+        borderColor: themeColors.borderFocus,
+    },
+    expensiPickerContainerDisabled: {
+        backgroundColor: colors.gray1,
+        color: colors.gray3,
+    },
+    expensiPickerLabel: {
+        fontSize: variables.fontSizeLabel,
+        color: themeColors.textSupporting,
+        fontFamily: fontFamily.GTA,
+        position: 'absolute',
+        left: 12,
+        top: 8,
+    },
+    expensiPicker: (disabled = false) => ({
+        iconContainer: {
+            top: 16,
+            right: 12,
+        },
+        inputWeb: {
```

Comment:
There's definitely some potential to DRY up `inputWeb`, `inputIOS`, and `inputAndroid`, maybe using some existing styles.

Reply:
Done